source/content/drupal-9.md

```diff
@@ -0,0 +1,161 @@
+
+<Alert title="Warning" type="danger">
+
+Currently, Drupal 9 is only available on Pantheon as a Early Access feature release. This means the feature is currently in active development and is intended for testing and evaluation purposes only. **Sites may be deleted without notice**.
+
+</Alert>
+
+Choose one of the following methods to get started with a new D9 site, to test your existing site's [compatibility with an upgrade](#test-an-existing-drupal-site-for-drupal-9-upgrade-compatibility), or to [test run your Pantheon Drupal 8 site on D9](#test-upgrade-an-existing-pantheon-drupal-8-site-to-drupal-9).
+
+## Create a New D9 Site with Integrated Composer
+
+Join our early access program to test out both D9 and Pantheon native Composer support.
+
+1. [Fill out this form](https://docs.google.com/forms/d/1lahWKMT2VHXfr9hg15VIQY2Kn6z_j77o7Te6hZqsNgw) to get access to our “Drupal 9 Early Access” group. Once you are added, you will access to a new "Drupal 9 Early Access" site creation option.
+
+1. Spin up a new "Drupal 9 Early Access" site with `ic-demo-2020-` as a site name prefix (i.e., `ic-demo-2020-beta3-d9-test`). This will give you [D9 starter codebase](https://github.com/stevector/drupal-9-project) and will automatically trigger a Composer install and `git commit` on site creation.
+
+## Create a D9 Site with Continuous Integration via Build Tools
+
+Want to regularly develop with continuous integration (CI) and automated tests? Leverage Pantheon’s [Build Tools](/guides/build-tools) to get up and running quickly with a CI workflow (CircleCI, BitBucket, GitLab), to build, test, and deploy D9.
+
+1. Install the [Terminus Build Tools plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin).
+   - This will require generating a Pantheon machine token, a CircleCI API token, and a GitHub API credential.
+1. Enable support for custom CircleCI Orbs in your Organizational settings. In CircleCI, click **Settings**, then **Security**, and click **Yes** to allow uncertified orbs:
+   ![Allow Uncertified Orbs in CircleCI](../images/circleci/circleci-allow-uncertified-orbs.png)
+1. Use Terminus to create a new D9 site, complete with a customizable CI workflow. Remember to replace `<NEW-SITE-NAME>` in this example with your new site name:
+
+  ```bash{promptUser: user}
+  terminus build:project:create stevector/drupal-9-with-pantheon-orb <NEW-SITE-NAME> --stability=dev
+  ```
+
+1. Clone your D8 [site’s codebase to your computer](https://pantheon.io/docs/local-development#get-the-code). You can create a new D8 site or use an existing D8 site:
+
+  ```bash{promptUser: user}
+  git clone <url-for-site-repo>
+  ```
+
+1. Use Terminus to create a [Multidev](/multidev) environment called `update-9.0.0-beta3` on your D8 site for testing:
+
+  ```bash{promptUser: user}
+  terminus multidev:create <site>.<env> update-9.0.0-beta3
```

Comment:
Multidev names can only be 11 characters long.

Reply:
Yes, and this is within a section that should be deleted entirely. See https://github.com/pantheon-systems/documentation/pull/5733#discussion_r428406329